tutorial/set-name.md

```diff
@@ -60,10 +61,10 @@ func (msg MsgSetName) ValidateBasic() sdk.Error {
 }
 ```
 
-`ValidateBasic` is used to provide some basic **stateless** checks on the validity of the `Msg`.  In this case, check that none of the attributes are empty. Note the use of the `sdk.Error` types here. The SDK provides a set of error types that are frequently encountered by application developers.
+`ValidateBasic` is used to provide some basic **stateless** checks on the validity of the `Msg`. In this case, check that none of the attributes are empty. Note the use of the `sdk.Error` types here. The SDK provides a set of error types that are frequently encountered by application developers.
 
 ```go
-// GetSignBytes Implements Msg.
+// GetSignBytes encodes the message for signing
```

Comment:
If you are making a change in the code in the `.md` files, please make a corresponding change in the `.go` file.